nsxiv.1

```diff
@@ -439,6 +424,20 @@ Color of the bar foreground. Defaults to window.foreground
 Color of the mark foreground. Defaults to window.foreground
 .TP
 Please see xrdb(1) on how to change them.
+.SH WINDOW TITLE
+The window title can be replaced with the output of a user-provided script,
+which is called by nsxiv whenever there's a redraw. The path of this script is
+.I $XDG_CONFIG_HOME/nsxiv/exec/win-title
+and the arguments given to it (where "Optional" arguments might be empty) are:
+1) full path to the current image,
+2) current file number,
+3) total file number,
```

Comment:
```suggestion
1) absolute path of the current image,
2) current image number,
3) total image count,
```

Reply:
Same issue with the `absolute` word.

Changing `file` -> `image` is fine by me. Though one thing to keep in mind, nsxiv doesn't check if all given to it are valid images or not for performance reasons. We only remove invalid files from the list when attempting to load/display them.

So the "total image" count may *actually* be, lets say 15, while the total file count is 30. In that case the script will receive 30 as it's 3rd param.

Reply:
Did not notice that. Then `file` is fine, though we may want to disclose the caveats as we do in `key-handler`.